pacupg/pacupg

```diff
@@ -79,38 +79,41 @@ aur(){
         warn "pacaur not found. Aborting\n"
         exit 1
     fi
-    source $pacaurconfig
-    [[ -z $clean || $clean != "true" ]] &&  
-        sudo sed -i '/clean=/d' $pacaurconfig && 
-        echo "clean=true" | sudo tee -a $pacaurconfig > /dev/null # Make sure clean option is off in pacaur
-
-    builddir=${builddir:-/tmp/pacaurtmp-$USER}
+    source "${pacaurconfig}"
+    [[ -z "${clean}" || "${clean}" != "true" ]] &&
+        sudo sed -i '/clean=/d' "${pacaurconfig}" &&
+        echo "clean=true" | sudo tee -a "${pacaurconfig}" > /dev/null # Make sure clean option is off in pacaur
 
+    builddir="${builddir:-${TMPDIR:-/tmp}/pacaurtmp-$USER}"
 
     msg "Checking for updates...\n"
-    aurpkgs=$(pacaur --color never $ignoreopts $devel --check | awk -F'aur  ' '{print $2}')
-    if [[ -z $aurpkgs ]]; then
+    aurpkgs="$(pacaur --color never ${ignoreopts} ${devel} --check | awk -F'aur  ' '{print $2}')"
+    if [[ -z "${aurpkgs}" ]]; then
```

Comment:
Neither quotes nor explicit notation is needed here. The whole reason bash added the `[[]]` construct was to deal with space-containing strings better. Also all -z does is test if it exists or not.